week-3/D-array-filter/exercise.js

```diff
@@ -7,8 +7,15 @@
 */
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
+var filterarr = [];
 
-var pairsByIndex; // Complete this statement
+pairsByIndexRaw.filter(function(item) {
+  if (item !== null && item !== false && item !== "whoops" && item.length > 1) {
```

Comment:
But what if pairsByIndexRaw contains other arbitrary strings or values like, "hello" or the number 3? Remember the "typeof" keyword might come in handy here for making your check more robust.